test/kafka-exactly-once/mzcompose.yml

```diff
@@ -0,0 +1,116 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+version: '3.7'
+
+mzworkflows:
+  kafka-exactly-once:
+    env:
+      KAFKA_VERSION: latest
+      # @Philip, how could I get a random seed in here?
+      SEED: ${SEED:-44}
+    steps:
+      - step: workflow
+        workflow: start-deps
+
+      - step: run
+        service: testdrive-svc
+        command: --seed ${SEED} --kafka-option=group.id=group1 before-restart.td
+
+      - step: kill-services
+        services: [materialized]
+
+      - step: start-services
+        services: [materialized]
+
+      - step: wait-for-mz
+        service: materialized
+        port: 6875
+
+      - step: run
+        service: testdrive-svc
+        command: --seed ${SEED} --no-reset --kafka-option=group.id=group2 after-restart.td
+
+        # TODO: @Philip, do I need this step?
```

Comment:
@philip-stoev Not sure I need this here.

Reply:
No, I do not think you need it. It is needed in tests that repeatedly restart kafka/zookeeper and the like and any data needs to be cleaned up between invocations.